output/slide-deck.qmd

```diff
@@ -33,7 +33,7 @@ institute:   |
       </td>
       <td>
         <right>
-          [![](<!--# TODO: UQ Logo -->){height="75"}](<!-- # TODO: UQ URL -->)
```

Comment:
(Thinking of adding also the SORTEE logo in between)

Reply:
I like that idea!